Dockerfile

```diff
@@ -71,8 +71,6 @@ RUN dpkg --add-architecture i386 \
     openjdk-8-jdk \
     patch \
     pkg-config \
-    python \
-    python-pip \
```

Comment:
Oh boy, we still have some Python2 leftovers...nice that you remove that :smile: